src/igvfd/audit/measurement_set.py

```diff
@@ -191,6 +191,15 @@ def audit_inconsistent_institutional_certification(value, system):
         audit_category: inconsistent institutional certification
```

Comment:
can the audit_detail say "human" somewhere, to make clear that the requirement is for human origin samples only?